techniques/failures/sticky-footer.html

```diff
@@ -0,0 +1,80 @@
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+		<head>
+			  <title>Failure of Success Criterion 2.4.12 Focus not obscured  due to a sticky footer hiding focused elements</title>
+		
+			  <link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+		
+		</head>
+		<body>
+		
+			<h1>Failure of Success Criterion 2.4.12 Focus not obscured  due to a sticky footer hiding focused elements</h1>
+		
+			<section id="meta">
+				<h2>Metadata</h2>
+				<p id="id"></p>
+				<p id="technology">failures</p>
+				<p id="type">failure</p>
+			</section>
+		
+			<section id="applicability">
+				<h2>Applicability</h2>
+					<p>Content using any technology which supports keyboard interfaces.</p>
+				</section>
+				<section id="description">
+				<h2>Description</h2>
+				<p>The objective of this failure is to avoid situations in which people using a keyboard interface (e.g. tabbing through links and controls) cannot see where the indicator is due to other authored content.</p>
+
+				<p>Any 'sticky' content that moves with the viewport can potentially obscure other elements on the page, including controls the user may tab to.</p>
+			</section>
+			
+			<section id="examples">
+				<h2>Examples</h2>
+		
+				<section class="example" id="example-1">
+					<h3>Sticky footer</h3>
+					<p>A Web page has a sticky footer, an element the width of the viewport and stays visible at the bottom of the viewport. The footer is tall enough to completely cover the element in focus as a user tabs down the page.</p>       
```

Comment:
The "and" clause in the first sentence reads awkwardly here, and I'm not sure the first part of it is necessary to the example anyway. Maybe something like this? (and similar update for the header example)

```suggestion
					<p>A Web page has a sticky footer, an element that stays visible at the bottom of the viewport as the user scrolls the page. The footer is tall enough to completely cover the element in focus as a user tabs down the page.</p>       
```